docs/artifacts/lockfile-blind-spot.html

```
@@ -0,0 +1,370 @@
+<title>The Lockfile Blind Spot</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)